core/trie/src/error.rs

```diff
@@ -12,22 +12,36 @@ use std::fmt;
 use std::error::Error as StdError;
 
 #[derive(Debug, PartialEq, Eq, Clone)]
-/// Error concerning the Parity-Codec based decoder.
+/// Error for trie node decoding.
 pub enum Error {
 	/// Bad format.
 	BadFormat,
```

Comment:
Is this still used?

Reply:
yes  core/trie/src/node_codec.rs  lines 73 for instance